docs/static-general-description.md

```diff
@@ -5,120 +5,145 @@ The format supports the following features
 
-- An Event can be sent from client to server or from server to client(\*).
-(\*) TODO: Shall the direction of an event be defined somehow?
+- \*\* Design tip:  If there is a need to model On/Off events, to avoid duplicating each one into two different Events -> give the event a single name and transfer a boolean as argument: FooState(bool) where the true/false value indicates the on/off state.  Another option might be to define an observable boolean data **Property** and use the target environment's Pub/Sub capability to get updates on its state.  (Depending on the details of the target environment, the latter might also give reliability guarantees that Events do not have).
 
-- \*\* If an event is to be transferred as a "broadcast" in a particular environment whereas other Events would be addressed uniquely to a particular client, then define this difference as extra information in a deployment model / target mapping.  It is not specified in the interface description.
+- Conceptually, an Event can be sent from client to server or from server to client(\*).
+(\*) TO CLARIFY: Shall the direction of an event be defined, and how?
+
+- \*\* If some events are to be transferred as a "broadcast" (from server to all clients) when other events would be addressed uniquely to particular client(s), then define this difference as extra information in a deployment model / target mapping.  It is not specified in the interface description.
```

Comment:
I do not really understand what the asterisks here are supposed to mean. Footnotes? If so difficult to follow.

Reply:
Yes, some sort of notes / related comments.
OK, I will try to improve this part of the text.

Reply:
Thanks for the review!